polynote-frontend/style/styles.less

```diff
@@ -1495,17 +1495,25 @@ button.inspect.icon-button {
     grid-area: head;
     position: relative;
     user-select: none;
+    display: flex; 
+    align-items: center; 
+    min-width: 0; 
+    overflow: hidden;
+    height: 1em;
+
     .left-buttons, .right-buttons {
-      position: absolute;
-      top: .1em;
-      bottom: .1em;
```

Comment:
Curious for the decision to remove these paddings - this a pretty fragile area of our CSS.

Reply:
Hi @omidmogasemi, to my knowlegde this part of css is only used in the header of notebook and kernel panel, and that's where we need flexbox layout to scale the search box properly 
-> removing absolut positioning from right button and make searchbox using the free space shifts  the right button again to the right.
As the buttons are no longer positioned absolute, in my opinion the alignment of the buttons has to just look nice again in end and be stable on different browsers and media queries. Thats what it does, see video below. Please check comment below to re-test on your side again...